beams/tree_config/prebuilt/wait_for_ack.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from dataclasses import dataclass, field
+
+import py_trees
+from py_trees.composites import Selector
+
+from beams.behavior_tree.action_node import ActionNode, wrapped_action_work
+from beams.behavior_tree.check_and_do import CheckAndDo
+from beams.tree_config.base import BaseItem
+from beams.tree_config.condition import AcknowledgeConditionItem
+from beams.typing_helper import Evaluatable
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class WaitForAckNodeItem(BaseItem):
+    ack_cond_item: AcknowledgeConditionItem = field(default_factory=AcknowledgeConditionItem)
+    wait_time_out: int = 60
+
+    def get_tree(self) -> Selector:
+
+        check_node = self.ack_cond_item.get_tree()
+
+        # check if node is acked
+        def check_acked() -> bool:
+            return check_node.check_ack()
+
+        # build waiting action node #TODO: probably put timeout here
```

Comment:
```suggestion
        # build waiting action node
```
There is a timeout?
Why is there two comment `#`'s?